databroker/headersource/shim.py

```diff
@@ -24,7 +24,11 @@ def __call__(self, text_search=None, filters=None, **kwargs):
         else:
             # Include empty {} here so that '$and' gets at least one query.
             _format_time(kwargs, self.mds.config['timezone'])
-            query = {'$and': [{}] + [kwargs] + filters}
+            formatted_filters = []
+            for v in filters:
+                _formatted_time(v, self.mds.config['timezone'])
```

Comment:
Also, notice that `filters` is also used in the text_search case above. The `filters` should be "formatted" above if/else statement that we're inside of here, so that both cases work.